app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: session_params[:email])
+    if user.present? && user.authenticate(session_params[:password])
+      render json: UserSerializer.new(user)
+    else
+      error_render
```

Comment:
Received a `DoubleRenderError` when the actions in `error_render` were in this method - moved it to a private method to get around this.